app/models/status.rb

```diff
@@ -270,6 +273,10 @@ def account_silencing_filter(account)
   end
 
   def local_only?
+    local_only
```

Comment:
ActiveRecord will generate a #local_only? predicate based on the existence of the local_only boolean column, so this definition can be omitted.